src/Field/Field.tsx

```diff
@@ -32,16 +33,25 @@ export const Field = ({
   required,
   showCaret,
   fullHeight = false,
+  assistiveText,
   ...marginProps
 }: FieldProps) => {
   return (
     <Container className={className} {...marginProps}>
       {label && (
         <Box mb={{ custom: outlined ? 4 : 0 }}>
           {renderAsTitle ? (
-            <Text tag="label" typo="heading-small" htmlFor={id} mb="16px">
-              {label}
-            </Text>
+            <Box mb="16px">
+              <Text tag="label" typo="heading-small" htmlFor={id}>
+                {label}
+              </Text>
+
+              {assistiveText && (
+                <Text tag="p" color="subtext">
```

Comment:
We should have 4px between title and assistive text

Reply:
resolved in [82e535](https://github.com/marshmallow-insurance/smores-react/pull/2118/commits/82e535d107f5efa170c85b575cc570d45c8c77aa)